src/components/page-layout.tsx

```diff
@@ -1,3 +1,4 @@
+import { AlertMessage } from "@fwl/web/dist/src/typings/utils";
```

Comment:
"@fwl/web/dist/src/typings/utils"

Is this path the right one ? 🤔

Reply:
It is. I also was confused by the path. It has to do with the way web exports its type declations. I'll double check thought, thanks for the reminder ;).

Reply:
I have found the issue. I'll wait for the next version of web to be published to update the version and the path here.